### PR TITLE
Simplify PHP installation

### PR DESCRIPTION
**Pull request by tomjn**
simplified PHP package installation to match closer to the main provisioner in VVV core, should be a little faster, with smaller output

### CODE REVIEW

**Comment by tomjn** [comment]
Improved the PHP provisioners some more, and added a general PHP provisioner that installs all available versions

File: php56/provision.sh [hunk 1]
```diff
@@ -72,72 +61,32 @@ network_check() {
...
-
 package_install() {
-  package_check
 
-  if [[ ${#apt_package_install_list[@]} = 0 ]]; then
-    echo -e "No apt packages to install.\n"
-  else
-
-    # Apply the PHP signing key
-    apt-key adv --quiet --keyserver "hkp://keyserver.ubuntu.com:80" --recv-key E5267A6C 2>&1 | grep "gpg:"
```
Comment (Mte90) [inline php56/provision.sh]: This command is disappeared, it will be executed in VVV instead?

Comment (Mte90) [reply php56/provision.sh]: Right now doesn't seems that is added https://github.com/Varying-Vagrant-Vagrants/VVV/search?q=apt-key&unscoped_q=apt-key

Comment (tomjn) [reply php56/provision.sh]: yes it is, we use Ondrej packages, which are added here:

https://github.com/Varying-Vagrant-Vagrants/VVV/blob/b3d1d0c8ca280142373b4ffb46ef3ccde4c369b7/provision/provision.sh#L379

**Pull request merged**